### PR TITLE
Update AdminLastMileFailuresTable.tsx

### PR DESCRIPTION
**Pull request by TomNUSDS**
Improve the modal content for the LastMileFailure Info Dialog.

Before:
image

After:
image

### CODE REVIEW

**Review by kevinhaube — COMMENTED** [review]
Is there any way to cut back on raw JSX? Not necessarily for reuse but just to keep file line numbers shorter

File: frontend-react/src/components/Admin/AdminLastMileFailuresTable.tsx [hunk 1]
```diff
@@ -78,6 +79,66 @@ ${data.receiver}`;
...
+                </Grid>
+
+                <Grid row className={"modal-info-row"}>
+                    <Grid className={"modal-info-label"}>Report ID:</Grid>
+                    <Grid className={`modal-info-value`}>
+                        {infoData.reportId}
+                    </Grid>
+                </Grid>
+
+                <Grid row className={"modal-info-row"}>
```
[the remark this answers is not in the document]
Comment (TomNUSDS) [reply frontend-react/src/components/Admin/AdminLastMileFailuresTable.tsx:123]: Sure. I can make it a data driven loop.

[the remark this answers is not in the document]
Comment (TomNUSDS) [reply frontend-react/src/components/Admin/AdminLastMileFailuresTable.tsx:123]: OK. I investigated. But there are a couple of unique entries  (`File URI` and `Info Details`) that prevent it from being turned into a clean loop. The resulting code is more complex and not that much sorter.

[the remark this answers is not in the document]
Comment (kevinhaube) [reply frontend-react/src/components/Admin/AdminLastMileFailuresTable.tsx:123]: Ah well, worth a shot!

**Review by kevinhaube — APPROVED** [review]
In light of above, LGTM!

**Comment by TomNUSDS** [comment]
@kevinhaube It was a legit callout to see if it was trivial to reduce duplicate react tsx. Thanks for the review!

**Pull request merged**